cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -279,7 +279,9 @@
     <name>app.program.extra.classpath</name>
     <value></value>
     <description>
-      Extra Java classpath for CDAP programs
+      Extra Java classpath for CDAP programs. Support wildcard suffix "*" to
+      include all jar files under a directory. The extra classpaths are not
```

Comment:
Rather than say these are not localized, I think it is clearer to say that this must be present on all nodes in your cluster. Something like:

Extra Java classpath for CDAP programs. Must be present on all nodes in the cluster. Supports...